packages/gatsby-source-shopify/src/create-schema-customization.ts

```diff
@@ -72,7 +72,7 @@ export function createSchemaCustomization(
     ownerType: `String!`,
     updatedAt: `Date!`,
     value: `String!`,
-    valueType: `String!`,
+    type: `String!`,
```

Comment:
We'll need both types here as well.
I think you can deprecate the valueType field by writing something like:
```
valueType: {
  type: `String!`,
  deprecationReason: `Shopify has deprecated this field`
}
```

https://graphql-compose.github.io/docs/api/Resolver.html#deprecationreason

Reply:
I should add that I'm not 100% sure those are the right properties there, I'm going off memory here 😅 but if it's not that it's very close to that

Reply:
they look right to me!

Reply:
Do they work when you run gatsby? You should be able to query both fields still

Reply:
Yeah we can still query both fields

Reply:
Sweet!